pkg/ddc/alluxio/ufs.go

```diff
@@ -25,7 +25,7 @@ func (e *AlluxioEngine) UsedStorageBytes() (value int64, err error) {
 	return e.usedStorageBytesInternal()
 }
 
-// return free storage size of Alluxio in bytes
+// FreeStorageBytes returns free storage size of Alluxio in bytes
```

Comment:
Could you please open an issue on it? thanks very much.

Reply:
Fixes #356.